test/keypair_test.exs

```diff
@@ -1,40 +1,48 @@
 defmodule KeyPairTest do
   alias Elixium.KeyPair
-  use ExUnit.Case, async: true
 
-  setup _ do
```

Comment:
Why aren't we clearing keys after running tests? Core doesn't need to save keys at all

Reply:
Keys clear now